From 6f544a5bd43446859754cb80e012af933b843db9 Mon Sep 17 00:00:00 2001
From: Florian Klink <flokli@flokli.de>
Date: Wed, 3 Jun 2020 22:05:34 +0200
Subject: [PATCH] clipboard: make which substitutable

This is used to detect the presence of xclip and other clipboard
handling tools.
---
 bpython/clipboard.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bpython/clipboard.py b/bpython/clipboard.py
index aee429b..f346429 100644
--- a/bpython/clipboard.py
+++ b/bpython/clipboard.py
@@ -58,7 +58,7 @@ class OSXClipboard(object):
 
 def command_exists(command):
     process = subprocess.Popen(
-        ["which", command], stderr=subprocess.STDOUT, stdout=subprocess.PIPE
+        ["@which@", command], stderr=subprocess.STDOUT, stdout=subprocess.PIPE
     )
     process.communicate()
 
-- 
2.26.2

